deploy/kiali/kiali_cr.yaml

```diff
@@ -277,6 +277,13 @@ spec:
 #    ---
 #    ingress_enabled: true
 #
+# Determines the logger configuration.
+# log_format supports text and json.
+# log_level supports trace, debug, info, warn, error, fatal.
+#    logger:
```

Comment:
To match/be consistent with the rest of the settings documented here, put "---" above this line (all settings groups have `---` to make it easier to find the actual settings). So this:
```suggestion
#    ---
#    logger:
```

Reply:
Also, all the peer settings here are sorted alphabetically. Can you sort so "log_level" is after "log_format".